resources/views/requests/show.blade.php

```diff
@@ -0,0 +1,129 @@
+
+                <div class="card-body">
+                    @if ($request->error_message)
+                        <div class="alert alert-danger" role="alert">
+                            <h6 class="alert-heading">Xebia Header Checker failed to execute the request</h6>
+
+                            {{$request->error_message}}
+                        </div>
+                    @endif
+
+                    <dl class="row">
+                        <dt class="col-sm-3">Url</dt>
+                        <dd class="col-sm-9">
+                            <a href="{{$request->endpoint->url}}" target="_blank">
+                                {{$request->endpoint->url}}
+                            </a>
+                        </dd>
+
+                        <dt class="col-sm-3">Method</dt>
+                        <dd class="col-sm-9">{{$request->endpoint->method}}</dd>
+
+                        <dt class="col-sm-3">Profile</dt>
+                        <dd class="col-sm-9">{{$request->profile->name}}</dd>
+                    </dl>
+
+                    @if ($request->requestHeaders->count() > 0)
+                        <h4>Request headers</h4>
+
+                        <dl class="row">
+                            @foreach ($request->requestHeaders as $requestHeader)
+                                <dt class="col-sm-3">{{$requestHeader->name}}</dt>
+                                <dd class="col-sm-9">{{$requestHeader->value}}</dd>
+                            @endforeach
+                        </dl>
+                    @endif
+                </div>
+            </div>
+        </div>
+    </div>
+
+    @foreach ($request->responses as $key => $response)
+        <div class="row">
+            <div class="col">
+                <div class="card">
+                    <div class="card-header">
+                        Response {{$key + 1}} information
+                    </div>
+
+                    <div class="card-body">
+                        <dl class="row">
+                            <dt class="col-sm-3">Status code</dt>
+                            <dd class="col-sm-9">{{$response->status_code}}</dd>
+
+                            <dt class="col-sm-3">Reason phrase</dt>
+                            <dd class="col-sm-9">{{$response->reason_phrase}}</dd>
+
+                            @if ($locationResponseHeader = $response->responseHeaders->where('name', 'Location')->first())
+                                <dt class="col-sm-3">Location</dt>
+                                <dd class="col-sm-9">
+                                    <a href="{{$locationResponseHeader->value}}" target="_blank">
+                                        {{$locationResponseHeader->value}}
+                                    </a>
+                                </dd>
+                            @endif
+                        </dl>
+
+                        <h4>Findings</h4>
+                        @if (count($response->findings) > 0)
```

Comment:
Aren't these collections? If so, you can do `->findings->isEmpty()` or `->findings->count()`. 😄

Reply:
`findings` is a array.